<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gpt_session_select</class>
 <widget class="QWidget" name="gpt_session_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select gpt session</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>77</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="view_btn">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_btn">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rename_btn">
      <property name="text">
       <string>Rename</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
